### PR TITLE
Census Household: Individual Repeating Section

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Added the repeating section for the householder.

Most of the individual section is same as the existing individual section, however, the proxy question on individual vs household is the opposite. So to minimise duplication, I made use of jsonnet's `jpath` option to add an additional library path depending on the schema type.

The folder structure has been changed to suit the new changes, hence this PR also effects the individual census schema.

### How to review 
Ensure:
- Changes are as per instructions in Trello.
- Individual census schema has not been altered. Check diff between this and v3 build.

[Quick Launch for GB-ENG](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3197-add-individual-repeating-section-eng-wales-ni/data/en/census_household_gb_eng.json)
[Quick Launch for GB-WLS](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-WLS&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3197-add-individual-repeating-section-eng-wales-ni/data/en/census_household_gb_wls.json)
[Quick Launch for GB-NIR](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3197-add-individual-repeating-section-eng-wales-ni/data/en/census_household_gb_nir.json)

Depends on: https://github.com/ONSdigital/eq-schema-validator/pull/157

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Add individual repeating section - Eng/Wales/NI (M)](https://trello.com/c/DPaUgnsa/3197-add-individual-repeating-section-eng-wales-ni-m)

**Review by JamesGardiner — APPROVED** [review]

**Review by JamesGardiner — DISMISSED** [review]
Looks like POT files need regenerating

File: data-source/jsonnet/northern-ireland/census_household.jsonnet [hunk 1]
```diff
@@ -1,31 +1,101 @@
...
-local self_contained = import 'blocks/household/accommodation/self_contained.jsonnet';
-local type_of_flat = import 'blocks/household/accommodation/type_of_flat.jsonnet';
-local type_of_house = import 'blocks/household/accommodation/type_of_house.jsonnet';
-local who_rent_from = import 'blocks/household/accommodation/who_rent_from.jsonnet';
+local accommodation_introduction = import 'household/blocks/accommodation/accommodation_introduction.jsonnet';
+local accommodation_summary = import 'household/blocks/accommodation/accommodation_summary.jsonnet';
+local accommodation_type = import 'household/blocks/accommodation/accommodation_type.jsonnet';
+local adapted = import 'household/blocks/accommodation/adapted.jsonnet';
+local central_heating = import 'household/blocks/accommodation/central_heating.jsonnet';
+local number_bedrooms = import 'household/blocks/accommodation/number_bedrooms.jsonnet';
```
Comment (ajmaddaford) [inline data-source/jsonnet/northern-ireland/census_household.jsonnet]: This file doesn't exist.

File: data-source/jsonnet/northern-ireland/census_household.jsonnet [hunk 2]
```diff
@@ -1,31 +1,101 @@
...
+local number_of_vehicles = import 'household/blocks/accommodation/number_of_vehicles.jsonnet';
+local own_or_rent = import 'household/blocks/accommodation/own_or_rent.jsonnet';
+local renewables = import 'household/blocks/accommodation/renewables.jsonnet';
+local self_contained = import 'household/blocks/accommodation/self_contained.jsonnet';
```
Comment (ajmaddaford) [inline data-source/jsonnet/northern-ireland/census_household.jsonnet]: This file doesn't exist.

File: data-source/jsonnet/northern-ireland/census_household.jsonnet [hunk 3]
```diff
@@ -1,31 +1,101 @@
...
+local nvq_level = import 'individual/blocks/qualifications/nvq_level.jsonnet';
+local other_qualifications = import 'individual/blocks/qualifications/other_qualifications.jsonnet';
+local qualifications = import 'individual/blocks/qualifications/qualifications.jsonnet';
+
+// Employment
+local armed_forces = import 'individual/blocks/employment/armed_forces.jsonnet';
+local business_name = import 'individual/blocks/employment/business_name.jsonnet';
+local employer_address_depot = import 'individual/blocks/employment/employer_address_depot.jsonnet';
+local employer_address_workplace = import 'individual/blocks/employment/employer_address_workplace.jsonnet';
+local employer_type_of_address = import 'individual/blocks/employment/employer_type_of_address.jsonnet';
```
Comment (ajmaddaford) [inline data-source/jsonnet/northern-ireland/census_household.jsonnet]: These three files don't exist.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
👏 

Just a few small file references to fix 🙂

File: data-source/jsonnet/northern-ireland/census_household.jsonnet (hunk 3 above)
Comment (MebinAbraham) [reply data-source/jsonnet/northern-ireland/census_household.jsonnet]: Copy and pasted from the individual schema. I have removed this reference from the individual.jsonnet as well.

File: data-source/jsonnet/northern-ireland/census_household.jsonnet (hunk 2 above)
Comment (MebinAbraham) [reply data-source/jsonnet/northern-ireland/census_household.jsonnet]: Ah, these were already existing, removed now. If only we had a linter 👀

File: data-source/jsonnet/northern-ireland/census_household.jsonnet (hunk 1 above)
Comment (MebinAbraham) [reply data-source/jsonnet/northern-ireland/census_household.jsonnet]: Ah, these were already existing, removed now. If only we had a linter 👀

**Review by ajmaddaford — APPROVED** [review]

File: data-source/jsonnet/england-wales/household/lib/rules.libsonnet [hunk 4]
```diff
@@ -0,0 +1,14 @@
+local common_rules = import '../../lib/common_rules.libsonnet';
+
+{
+  isNotProxy: {
```
Comment (bitdivision) [inline data-source/jsonnet/england-wales/household/lib/rules.libsonnet:4]: :+1:  Much better name

**Review by donnap573 — APPROVED** [review]

**Pull request merged**